Dockerfile-custom

```diff
@@ -0,0 +1,9 @@
+FROM localhost/maestro-base:latest
+
+# COPY <src> agent.yaml      # copy agent file into image
+# COPY <dest> workflow.yaml  # copy workflow file into image
```

Comment:
I would mount these as a volume at runtime
as an example of customization, maybe show to add other python libraries (e.g. pip install ...)